pkg/dashboard/ttlcontroller/fx.go

```diff
@@ -0,0 +1,24 @@
+// Copyright Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ttlcontroller
+
+import (
+	"github.com/chaos-mesh/chaos-mesh/pkg/dashboard/core"
+	"github.com/go-logr/logr"
+)
+
+func InitInject(logger logr.Logger, experiment core.ExperimentStore, event core.EventStore, schedule core.ScheduleStore, workflow core.WorkflowStore, ttlc *TTLconfig) *Controller {
```

Comment:
There is a typo in the type of the last parameter, it should be `TTLConfig`.

And I prefer to use `Bootstrap` as the function name, with keeping the same style with other methods provided to `fx`

Rest LGTM!

Reply:
` TTLconfig` seems to be correct here, I don't know why it's showing undefined.
😕

Reply:
> with keeping the same style with other methods provided to fx

do you mean I need to update the function to as below?
```go
func Bootstrap(conf *config.ChaosDashboardConfig, workflow core.WorkflowStore, logger logr.Logger) *Controller {
.
.
}
```

Reply:
> ` TTLconfig` seems to be correct here, I don't know why it's showing undefined. confused

It should be `TTLConfig` not `TTLconfig`, notice the capital character "C" in "Config"

Reply:
> > with keeping the same style with other methods provided to fx
> 
> do you mean I need to update the function to as below?
> 
> ```go
> func Bootstrap(conf *config.ChaosDashboardConfig, workflow core.WorkflowStore, logger logr.Logger) *Controller {
> .
> .
> }
> ```

Yes. I saw your commits, you did great. :D